serialization/Schemas/spdx-v3.json

```diff
@@ -0,0 +1,1061 @@
+  "properties": {
+    "payload": {
+      "$ref": "#/definitions/Payload"
+    },
+    "elements": {
+      "$ref": "#/definitions/Elements"
+    }
+  },
+  "definitions": {
+    "Payload": {
+      "title": "Payload",
+      "type": "object",
+      "description": "Serialized file (or equivalent) for transferring Elements",
+      "additionalProperties": false,
+      "required": [
+        "namespace",
+        "created",
+        "creator",
+        "specVersion",
+        "profile",
+        "dataLicense",
+        "element"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "namespace": {
+          "$ref": "#/definitions/IRI",
+          "description": "Default namespace for Element ids in this file (rdf BASE)"
+        },
+        "prefixes": {
+          "$ref": "#/definitions/NamespaceMap",
+          "description": "Namespace abbreviations used in this file (rdf PREFIX)"
+        },
+        "created": {
+          "$ref": "#/definitions/DateTime",
+          "description": "File creation info and element defaults"
+        },
+        "creator": {
+          "type": "array",
+          "description": "File creation info and element defaults - Link(Identity)",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Element",
+            "description": "File creation info and element defaults - Link(Identity)"
+          }
+        },
+        "specVersion": {
+          "$ref": "#/definitions/SemVer",
+          "description": "File creation info and element defaults"
+        },
+        "profile": {
+          "type": "array",
+          "description": "File creation info and element defaults",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/ProfileIdentifier",
+            "description": "File creation info and element defaults"
+          }
+        },
+        "dataLicense": {
+          "$ref": "#/definitions/LicenseId",
+          "description": "File creation info and element defaults"
+        },
+        "element": {
+          "type": "array",
+          "description": "Element values, defined in this file or copied",
+          "uniqueItems": true,
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Element",
+            "description": "Element values, defined in this file or copied"
+          }
+        },
+        "spdxDocumentId": {
+          "$ref": "#/definitions/Element",
+          "description": "Optional SpdxDocument element that describes this file"
+        }
+      }
+    },
+    "Elements": {
+      "title": "Elements",
+      "type": "array",
+      "description": "Serialized set of independent Element values",
+      "uniqueItems": true,
+      "minItems": 1,
+      "maxItems": 100,
+      "items": {
+        "$ref": "#/definitions/Element"
+      }
+    },
+    "NamespaceMap": {
+      "title": "NamespaceMap",
+      "type": "object",
+      "additionalProperties": false,
```

Comment:
`NamespaceMap` has the two properties `prefix` and `namespace` according to the model.png

Reply:
That's a bug in my tooling - thanks for catching that!

The source in `serialization/Schemas/spdx-v3.jidl` is:
```
NamespaceMap = MapOf(Prefix, IRI){1..*}
```
which means that every property name must be a Prefix (without explicitly enumerating them) and every value is an IRI.  Now I'm thinking your validation errors mentioned earlier are also errors in my JSON Schema generation tool.